src/leaderboard.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import {PointsService} from "./services/PointsService";
+function Leaderboard({isAuthenticated, user, login, logout }) {
+
+    const [leaderboardsList, setLeaderboardsList] = useState([]);
+    const [message, setMessage] = useState('');
+
+    useEffect(() => {
+        const fetchLeaderboardsData = async () => {
+            try {
+                const gl = await PointsService.getLeaderboards();
+                setLeaderboardsList(gl.data.data);
+            } catch (e) {
+                setMessage('There was a problem with fetching leaderboards data.');
+            }
+        };
+
+        fetchLeaderboardsData();
+    }, []); 
+
+    return (
+        <div className="list">
```

Comment:
potrzebna jest tabelka, albo przynajmniej ramka wokół każdego wpisu, żeby to było czytelne